docs/api.html

```diff
@@ -16,7 +16,7 @@
 				<li>There are changes to the Node API.  See the README for details.</li>
 			</ul>
 
-			<h2>GET /v{{apiversion}}/polyfill<var>:minify</var>.<var>:type</var></h2>
+			<h2>GET /v2/polyfill<var>:minify</var>.<var>:type</var></h2>
```

Comment:
Although the docs are versioned, we currently only support v2, so this makes sense,but we should retain the ability to have separate docs for v2 and a potential v3.